Sources/TokamakDemo/main.swift

```diff
@@ -23,10 +23,22 @@ _ = document.head.object!.insertAdjacentHTML!("beforeend", #"""
   href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
 """#)
 
+struct CustomModifier : ViewModifier {
+  func body(content: Content) -> some View {
+    Text("Whole new body!")
+  }
+}
+
 let div = document.createElement!("div").object!
 let renderer = DOMRenderer(
   VStack {
     Counter(count: 5, limit: 7)
+    ZStack {
+      Text("I'm on bottom")
+      Text("I'm on top")
```

Comment:
Please also add a third view that uses the `zIndex` modifier directly to demo that as well.

Reply:
`zIndex` currently does not work as mentioned above. Did you have any thoughts on the view metadata, or have another idea of how to expose this?

I could possibly make the `zIndex` modifier wrap the modified view. Then in the `ZStack` `deferredBody` check if the View is that wrapper, and apply the z-index that way.

Reply:
Ah, sorry I missed that, having a closer look now...